posthog/models/team/team.py

```diff
@@ -197,21 +199,22 @@ def get_effective_membership_level(self, user_id: int) -> Optional["Organization
 
     @property
     def actor_on_events_querying_enabled(self) -> bool:
-        # Two part gate to ensure we don't accidentally enable this for everyone prematurely.
-        # Nor should we start making excessive requests to the API to get results.
-        # Temporary, until we're convinced the library works well enough.
-        enabled_teams = get_list(get_instance_setting("ENABLE_ACTOR_ON_EVENTS_TEAMS"))
-        if len(enabled_teams) > 0:
-            return str(self.pk) in enabled_teams or "all" in enabled_teams
-
-        # only when instance setting list is empty, use the feature flag
-        return posthoganalytics.feature_enabled(
-            "person-on-events-enabled",
-            str(self.uuid),
-            groups={"project": str(self.uuid)},
-            group_properties={"project": {"id": str(self.pk)}},
-            only_evaluate_locally=True,
-        )
+        # If the async migration is not complete, never enable actor on events querying.
+        if not person_on_events_ready():
```

Comment:
Consider excluding this check for `MULTY_TENANCY` we might be messing with the async migration on cloud and also no need to look it up every release from the async migrations table.

Reply:
Makes sense